<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerBreakpointsBrowserWidgetForm</class>
 <widget class="QWidget" name="SeerBreakpointsBrowserWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="breakpointsTreeWidget">
     <property name="columnCount">
      <number>12</number>
     </property>
     <column>
      <property name="text">
       <string>Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Disposition</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Enabled</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Function</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fullname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Thread Groups</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Times</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Original Location</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="refreshBreakpointsToolButton">
       <property name="toolTip">
        <string>Refresh the list of breakpoints.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/view-refresh.svg</normaloff>:/seer/resources/RelaxLightIcons/view-refresh.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="addBreakpointToolButton">
       <property name="toolTip">
        <string>Add a new breakpoint.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/document-new.svg</normaloff>:/seer/resources/RelaxLightIcons/document-new.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteBreakpointsToolButton">
       <property name="toolTip">
        <string>Delete selected breakpoints.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/edit-delete.svg</normaloff>:/seer/resources/RelaxLightIcons/edit-delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="enableBreakpointsToolButton">
       <property name="toolTip">
        <string>Enable selected breakpoints.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-add.svg</normaloff>:/seer/resources/RelaxLightIcons/list-add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="disableBreakpointsToolButton">
       <property name="toolTip">
        <string>Disable selected breakpoints.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</normaloff>:/seer/resources/RelaxLightIcons/list-remove.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
